### PR TITLE
Skylark Upload Limit

### PR DESCRIPTION
**Pull request by mfine**
Master version of https://github.com/swift-nav/piksi_buildroot/pull/216.

/cc @denniszollo @jacobmcnamee

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## v1.1.0-branch-72-gb99cc25
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.0-branch-72-gb99cc25/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.1.0-branch-72-gb99cc25/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.0-branch-72-gb99cc25

File: package/sbp_protocol/src/sbp_router.yml [hunk 1]
```diff
@@ -30,7 +30,9 @@ ports:
           - { action: ACCEPT }
       - dst_port: SBP_PORT_SKYLARK
         filters:
-          - { action: ACCEPT }
+          - { action: ACCEPT, prefix: [0x55, 0xOA, 0x02] } # Publish: SBP MSG_POS_LLH
```
Comment (jacobmcnamee) [inline package/sbp_protocol/src/sbp_router.yml]: That looks like an 'O' instead of a '0'.

Comment (mfine) [inline package/sbp_protocol/src/sbp_router.yml]: nice catch

**Comment by anonymous-travis-swiftnav** [comment]
## v1.1.0-branch-73-g76d4681
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.0-branch-73-g76d4681/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.1.0-branch-73-g76d4681/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.0-branch-73-g76d4681

**Comment by mfine** [comment]
Bringing this into master.

**Pull request merged**